backend/src/impl/benchmark_utils.py

```diff
@@ -156,22 +158,47 @@ def aggregate_view(
         input_df: pd.DataFrame, view_spec: BenchmarkViewConfig
     ) -> pd.DataFrame:
         output_df = input_df.copy()
+        skip_systems = False
         for operation in view_spec.operations:
-            group_by = ["system_name"] + operation.get("group_by", [])
-            if operation["op"] == "mean":
-                output_df = output_df.groupby(group_by).mean()
-            elif operation["op"] == "multiply":
-                output_df["score"] = output_df["score"] * output_df[operation["other"]]
-            elif operation["op"] == "sum":
-                output_df = output_df.groupby(group_by).sum()
+            skip_systems = operation["skip_groupby_system"]
+            group_by = ["system_name"] + [operation.get("group_by", [])]
```

Comment:
Or even better:
`group_by  = ([] if skip_systems else ["system_name"]) + operation.get("group_by", [])`

Reply:
I don't know if this is an issue with the version of Python I'm using or something, but for me operation.get actually didn't return a list as desired. Therefore, I had to wrap it square brackets to force it. However, even if I did 
group_by = ([] if skip_systems else ["system_name"]) + operation.get("group_by", [])
I'm afraid we will run into issues, because if there is no "group_by" then [operation.get("group_by",[])] will return [[]], which is also not good :(

Reply:
Yep, done! :D